Fix Database 2 name in readme

tenth10th opened the pull request:
The database inside the "Database 2" instance (on port 5433) seems to be called `message_store`?

juanpaco commented:
I  must have my GitHub notifications messed up somehow.  I only saw this today.  Thank you for the contribution!

tenth10th committed:
Fix Database 2 name in readme

The database inside the "Database 2" instance (on port 5433) is  called `message_store`

juanpaco reviewed (approved):
Indeed.  Copy pasta for the win.  Thanks for catching this!
